src/app/public/modules/phone-field/phone-field.component.ts

```diff
@@ -365,8 +365,16 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
       let foundCountry = this.countries
         .find(country => country.dialCode === dialCode && country.priority === 0);
 
-      if (foundCountry && foundCountry !== this.selectedCountry) {
-        newCountry = foundCountry;
+      if (foundCountry) {
+        // Ensure that the country that was found in one of our supported countries
```

Comment:
Did you mean to say, "...that was found _is_ one of our supported countries"?

Reply:
Ya. I slightly tweaked the end of that sentence.